app/Main.hs

```diff
@@ -25,19 +28,14 @@ import Backend
 import Coffer.Path ( Path, EntryPath )
 import qualified Entry as E
 import qualified Coffer.Directory as Dir
-
-url :: BaseUrl
-url = BaseUrl Http "127.0.0.1" 8200 ""
-
-token :: VaultToken
-token = VaultToken "s.AnHJuzOxRiD12eLxALFytccC"
-
-mount :: Text
-mount = "secret"
-
-runVaultIO_ :: Sem '[BackendEffect, Error CofferError, Embed IO, Final IO ] a -> IO a
-runVaultIO_ action =
-  runVaultIO url token mount action
+import Config (configCodec, Config (..))
+
+runVaultIO_ 
```

Comment:
I guess this function has now been generalized to work with _any_ backend (not necessarily vault's backend), so maybe we should rename it to something more generic like `runBackend`.

Reply:
Done